.chloggen/remove_metrics_proxy.yaml

```
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
+
+# The name of the component, or a single word describing the area of concern, (e.g. tempostack, tempomonolithic, github action)
+component: tempostack, tempomonolitic
```

```suggestion
component: operator
```

this change affects the metrics of the operator, not individual operand instances (correct me if I'm wrong)